README.md

```diff
@@ -6,7 +6,136 @@ This package contains methods providing Saleor business logic for a storefront a
+
+### Using React
+
+First install SDK as dependency to your project
+
+```bash
+npm install @saleor/sdk
+```
+
+Use `SaleorProvider` with passed Saleor's client created by `createSaleorClient` in a prop. Then use React hooks in any component passed as child to `SaleorProvider`.
+
+```tsx
+import { SaleorProvider, createSaleorClient, useAuth, useAuthState } from "@saleor/sdk";
+
+const client = createSaleorClient({
+  apiUrl: "http://localhost:8000/graphql/,
+  channel: "",
+});
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(
+  <SaleorProvider client={client}>
+    <App />
+  </SaleorProvider>,
+  rootElement
+);
+
+const App = () => {
+  const { login } = useAuth();
+  const { authenticated, user } = useAuthState();
+
+  const handleSignIn = async () => {
+    const { data } = await login({email: "admin@example.com", password: "admin"});
+
+    if (data.tokenCreate.errors.length) {
+      /**
+       * Unable to sign in.
+       **/
+    } else if (data) {
+      /**
+       * User signed in successfully.
+       **/
+    }
+  };
+
+  if (authenticated && user) {
+    return <span>Signed in as {user.firstName}</span>;
+  } else {
+    return <button onClick={handleSignIn}>Sign in</button>;
+  }
+};
+```
+
+### Using with NodeJS and other frameworks
+
+```bash
+npm install @saleor/sdk
+```
+
+Then use `createSaleorClient` to get Saleor api methods and internal config variables like channel and Apollo client.
+
+```tsx
+import { createSaleorClient } from "@saleor/sdk";
+
+const client = createSaleorClient({
+  apiUrl: "http://localhost:8000/graphql/,
+  channel: "",
+});
+
+const { auth, config, _internal } = client;
+```
+
+Finally, API methods can be used:
+
+```tsx
+const { data } = await auth.login({
+  email: "admin@example.com",
+  password: "admin",
+});
+
+if (data.tokenCreate.errors.length) {
+  /**
+   * Unable to sign in.
+   **/
+} else if (data) {
+  /**
+   * User signed in successfully.
+   **/
+  const userData = api.auth.tokenCreate.user;
+}
+```
+
+## Features
+
+We provide an API with methods and fields, performing one, scoped type of work. You may access them straight from `createSaleorClient()` or use React hooks, depending on [which setup do you select](#setup).
+
+| API object | React hook                            | Description                                                                     |
+| :--------- | :------------------------------------ | :------------------------------------------------------------------------------ |
+| `auth`     | `useAuth()`, `useAuthState()`         | Handles user authentication and stores data about the currently signed in user. |
+| `user`     | `useUser()`                           | Handles user account methods.                                                   |
+| `checkout` | `useCheckout()`, `useCheckoutState()` | Handles the whole checkout process.                                             |
```

Comment:
This doesn't exist yet

Reply:
We have pending first checkout pull request, checkout will be available with first official release and lets leave it in readme, not to update it again.

Reply:
Remove this for now, apart from it all looks good 🆗